site/mkdocs.yml

```diff
@@ -79,6 +79,7 @@ markdown_extensions:
 
 extra:
   icebergVersion: '1.5.0'
+  nessieVersion: '0.77.1'
```

Comment:
NessieVersion was not resolved in 
https://iceberg.apache.org/docs/latest/nessie/#spark-sql-extensions

Reply:
Great catch!